Tests/NIOSSLTests/SSLCertificateTest.swift

```diff
@@ -480,6 +480,9 @@ class SSLCertificateTest: XCTestCase {
     }
 
     func testNotValidAfter() throws {
+        guard MemoryLayout<time_t>.size >= 8 else {
+            throw XCTSkip()
+        }
```

Comment:
optional nit: this is now the behaviour we want but we can make the spelling a bit more idiomatic by using `XCTSkipUnless` and include a reason:
```suggestion
        try XCTSkipUnless(MemoryLayout<time_t>.size >= 8, "size of time_t must be 64bit or greater")
```